### PR TITLE
Release notes for July 18

### PR DESCRIPTION
**Pull request by gboone**
Merge in the morning to create the release notes for the latest IP sprint!

### CODE REVIEW

File: content/updates/2017-07-18-release-notes.md [hunk 1]
```diff
@@ -15,12 +15,12 @@ Curious what’s new that you might find helpful as a cloud.gov user? Here are h
...
 
 ### Platform releases
-We upgraded the Cloud Foundry deployment to [v267](https://github.com/cloudfoundry/cf-release/releases/tag/v264).
+We upgraded the Cloud Foundry deployment to [v267](https://github.com/cloudfoundry/cf-release/releases/tag/v267).
 
 You should restage your application to incorporate fixes in the base filesystem and ensure you’re running the most recent language version supported by your buildpack.
 
 #### Additional upgrades
-* [RootFS cflinuxfs2 1.134.0](https://github.com/cloudfoundry/cflinuxfs2/releases/tag/1.138.0), which addresses this security vulnerability:
+* [RootFS cflinuxfs2 1.134.0](https://github.com/cloudfoundry/cflinuxfs2/releases/tag/1.134.0), which addresses this security vulnerability:
```
Comment (brittag) [inline content/updates/2017-07-18-release-notes.md]: Not sure if this was supposed to be corrected to 1.134.0 or if the text was supposed to be corrected to to 1.138.0.

**Comment by brittag** [comment]
@gboone Put in a few fixes - can you take a look and see if they're the right fixes? If so, I can approve this and you can merge it!

File: content/updates/2017-07-18-release-notes.md (hunk 1 above)
Comment (gboone) [reply content/updates/2017-07-18-release-notes.md]: I will double check this with the slides. Good catch.

**Comment by gboone** [comment]
OK, I fixed up the RootFS version and the other fixes look good to me. Thanks @brittag!

**Review by brittag — APPROVED** [review]

**Pull request merged**